addons/mail/static/src/discuss/core/web/channel_selector.js

```diff
@@ -99,7 +100,7 @@ export class ChannelSelector extends Component {
                 choices.push({
                     channelId: "__create__",
                     classList: "o-discuss-ChannelSelector-suggestion",
-                    label: cleanedTerm,
+                    label: unaccent(this.state.value, true),
```

Comment:
Why `unaccent`? It looks like what we want is to strictly keep user input for channel name.